src/coreclr/hosts/unixcoreconsole/CMakeLists.txt

```diff
@@ -31,8 +31,4 @@ if(NOT CLR_CMAKE_PLATFORM_ANDROID)
   )
 endif()
 
-add_dependencies(coreconsole
-    coreclr
```

Comment:
@rahku Thanks for explanation. Then cmake traverses most of CMakeLists.txt, but finally corerun and coreconsole is not build, right?